packages/docs/stories/templates/form.stories.tsx

```diff
@@ -0,0 +1,182 @@
+import React from 'react';
```

Comment:
🎉

Reply:
:) Needed for the docs renderer unfortunately, as Storybooks config does not respect the  "Import not Needed" functionality of react@18 :(